.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Run tests
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Upgrade pip
+        run: python -m pip install --upgrade pip setuptools
+      - name: Install dependencies
+        run: pip install black isort mypy pylint pytest tutor types-docutils types-PyYAML types-setuptools
+      - name: Format with Black
+        run: make format
+      - name: Sort imports with isort
+        run: make isort
+      - name: Python tests
```

Comment:
This is not just Python tests, but all tests. You should change the name of that step to reflect that.

Reply:
do you have any particular name in mind?

Reply:
Changed to Test lint, types, and format